package.json

```diff
@@ -70,7 +70,7 @@
     "typescript": "2.8.1"
   },
   "dependencies": {
-    "@kubernetes/typescript-node": "0.1.1",
+    "@kubernetes/client-node": "0.6.0",
```

Comment:
will you check in the `package-lock.json` file?

Reply:
just checked in